zokrates_parser/src/zokrates.pest

```diff
@@ -84,7 +86,8 @@ spread_or_expression = { spread | expression }
 range_or_expression = { range | expression }
 array_initializer_expression = { "[" ~ expression ~ ";" ~ constant ~ "]" }
 
-unary_expression = { op_unary ~ term }
+signeable_expression = { "(" ~ expression ~ ")" | primary_expression }
+exponent_expression = { "(" ~ expression ~ ")" | primary_expression }
```

Comment:
`signeable_expression` is redundant with 'exponent_expression' and not referred to elsewhere in the grammar. Why is it needed?

Reply:
I thought I removed that! Good shout!

Reply:
It is not and I actually removed it just before you posted this :p

Reply:
I don't know why this doesn't show as outdated, maybe have a look at the latest diff here and close this if you're cool!